### DISCUSSION TITLE
What kinds of Coordination Tasks/Topics do you want to see across Science Centers?

### DISCUSSION THREAD

**Discussion by shannonrankin** [SAEL-SWFSC/PAM-SI-Mtg-March2024#78] [Q&A]
What kinds of task/topics do you want to coordinate across Science Centers?

Open to all ideas or thoughts.... 
If someone puts an idea you like a lot, add an emoji or up arrow.

**Comment by sofievanparijs** [comment]
Github task team

**Comment by GenevieveDavis** [comment]
Analysis protocols

**Reply by sofievanparijs** [reply]
Including viewer software(s)  - what is available for detections

**Reply by mcastellot** [reply]
and the tools asociated to these protocols if these are custom made

**Reply by sahav** [reply]
Follow SoundCoop archiving protocols for SI soundscape tasks

**Comment by mcastellot** [comment]
"Fully" annotated datasets for ML training

**Reply by sofievanparijs** [reply]
List of who already has annotated data sets and for what from different science centers

**Reply by GenevieveDavis** [reply]
With annotations in same formats

**Reply by mcastellot** [reply]
Do not just limit this to species, let's consider anthopogenic noise sources, and environmental sources too (rain, waves, thunder, ice, etc.) if these have been annotated by anybody

**Comment by shannonrankin** [comment]
Share progress & info related to PAM-SI task teams (Glider, NCEI, Cloud, Analytics)

**Comment by AnnAllen-NOAA** [comment]
Modification of analysis (triton, pamguard, etc) and data processing code to read/write flac compression format (where feasible) to reduce data archive

**Reply by shannonrankin** [reply]
Moving to Detection/Analysis methods for this afternoon

**Comment by ybarkley** [comment]
Identify what aspects of PAM can be standardized and incorporated into stock assessments for each group. Might not be density or abundance, but are there low-hanging fruits we can tackle together?

**Reply by shannonrankin** [reply]
Will add this to Stock Assessment and Climate groups to consider: "What topics/tasks do you need help sharing (Github/Open Science Group)"

**Comment by marlaholt** [comment]
consider using @ everyone to tag people for important questions that we need to respond to
